export async function fetchUsers(): Promise<User[]> {
    try {
        const response = await fetch("/users/api");
        if (!response.ok) {
            throw new Error("Erro ao buscar usu치rios");
        }
        return await response.json();
    } catch (error) {
        console.error("Erro ao buscar usu치rios", error);
        return [];
    }
}

export async function deleteUserFromAPI(id: number): Promise<void> {
    try {
        const response = await fetch(`/users/${id}`, { method: "DELETE" });
        if (!response.ok) {
            throw new Error("Erro ao excluir usu치rio");
        }
    } catch (error) {
        console.error("Erro ao excluir usu치rio:", error);
    }
}
